skipchain/struct.go

```diff
@@ -363,21 +397,32 @@ func (bl *BlockLink) VerifySignature(publics []kyber.Point) error {
 
 // SkipBlockMap holds the map to the skipblocks. This is used for verification,
 // so that all links can be followed.
+// TODO remove when scmgr is updated
```

Comment:
Why can't this be done now?

Reply:
Discussed this with @ineiti. Since he's changing the config structure in scmgr, we wanted it to make it easier to merge when his PR is done. That is, instead of merging both skipchain and scmgr at once, we plan to merge them separately.